### PR TITLE
Guesh 07 bnf abb 114

### PR DESCRIPTION
**Pull request by GueshSol**

### CODE REVIEW

File: ParisBNF/abb/BNFabb114.xml [hunk 1]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        </desc> 
+                     </decoNote>
+
+                     <decoNote type="miniature" xml:id="d11">
+                        <locus from="25vb" to="26r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1116MMJerusalem"/> typically in four episodes/scenes,
+                           showing from the bottom left to the right: first, the sick pilgrim lying
+                           in bed; second the sick pilgrim standing before the places to which <persName ref="PRS6819Mary"/>
+                           carries him; third, in varying sequence Golgotha, Jerusalem, the mount of
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1021]: the same is true for "varying" as for "typically": This only makes sense in the art theme, here, there must be one concrete sequence

File: ParisBNF/abb/BNFabb114.xml [hunk 2]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           episodes/events, showing: first, <persName ref="PRS4587Georget"/> lying
+                           in prison bound at his hands and feet, with a dove appearing to him;
+                           second, <persName ref="PRS4587Georget"/> standing healed from his
+                           wounds.
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d22">
+                        <locus target="#50v"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1127MMQatir"/>typically with sequence of three
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1144]: corresp="AT1127MMQatir"/>typically  space missing

File: ParisBNF/abb/BNFabb114.xml [hunk 3]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           man showing his healed foot to the bishop with the people standing in the
+                           background.
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d25">
+                        <locus from="58va" to="59r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1130MMMarqorewos"/>typically with sequence of seven
+                           episodes/events, showing: first, the leprous bishop Marqorewos standing
+                           before Patriarch Zakārǝzās and showing him his leprosy; second, the
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1184]: Zakārǝyās

File: ParisBNF/abb/BNFabb114.xml [hunk 4]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           patriarch warning Merqorewos not to deliver liturgical service; third,
+                           Marqorewos praying before the icon of the <ref type="authFile"
+                              corresp="AT1031VirginAndChild"/>; fourth, <ref type="authFile"
+                                 corresp="AT1031VirginAndChild"/> healing the leprous bishop and performing gesture of blessing; fifth,
+                           the bishop showing himself healed to patriarch Zakārǝzās; sixth, the bishop showing himself healed to his student ; and
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1189]: Zakārǝyās

File: ParisBNF/abb/BNFabb114.xml [hunk 5]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                 type="authFile" corresp="AT1031VirginAndChild"/> on another episode; fourth, the custodian
+                           looking the broken woman who is standing near to him with the icon of <ref
+                              type="authFile" corresp="AT1031VirginAndChild"/>  performing a gesture of blessing from the other side.
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d27">
+                        <locus from="65v" to="66r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1132MMSophia"/>typically with sequence of seven
+                           episodes/events, showing: first, Sofyā lying on top of a building before
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1209]: decide whether you want to use Sophia or Sofyā in the description

File: ParisBNF/abb/BNFabb114.xml [hunk 6]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d29">
+                        <locus target="#70v"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1134MMAnestasyos"/> typically with sequence of two
+                           episodes/events, showing: first, ʾAnǝsṭāsyos standing and praying in a church
+                           where the icon of the <ref type="authFile" corresp="AT1031VirginAndChild"/> depicted in front of him; second,ʾAnǝsṭāsyos lying
+                           before the icon of <ref type="authFile" corresp="AT1031VirginAndChild"/>, who is extending her hand towards him and announcing him good news
+                           cncerning his illness.
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1241]: concerning

File: ParisBNF/abb/BNFabb114.xml [hunk 7]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d30">
+                        <locus from="71v" to="72r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AAT1135MMSalusi"/> typically with sequence of four
+                           episodes/events, showing: first, the monk Sālusi surrounded by other monks,
+                           who are attacking him; second, the monks preventing him for eating;
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1249]: preventing him to eat?

File: ParisBNF/abb/BNFabb114.xml [hunk 8]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           Sālusi stood up near to the Church after penetrating the wall of the
+                           Church with his skull-cap fallen to the ground. 
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d31">
+                        <locus from="74v" to="75r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1136MMCannibal"/> typically with sequence of eight
+                           episodes/events, showing: first, the cannibal surrounded by the bodies of
+                           those he killed; Second the cannibal asking the peasant to sell his ox;
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1261]: second

File: ParisBNF/abb/BNFabb114.xml [hunk 9]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+
+
+                     <decoNote type="miniature" xml:id="d32">
+                        <locus target="#77r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1137MMThief"/>typically with sequence of four
+                           episodes/events, showing: first, the widow's son stealing cloth from a
+                           church; second, members of the Church arresting and starting to punish
+                           the thief; third, <persName ref="PRS6819Mary"/> ordering the punishers to
+                           let him free performing gesture of blessing; and fourth, the thief
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1283]: a gesture of blessing, here and throughout

File: ParisBNF/abb/BNFabb114.xml [hunk 10]
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                              corresp="AT1138MMFlood"/>typically with as a sequence of three
+                           episodes/events, showing: first, a church amongst the waves from which
+                           the people are fleeing; second, the woman holding her new-born child
+                           protected by <persName ref="PRS6819Mary"/> from the water; third, the
+                           woman with her child surrounded by the marvelling people.
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d34">
+                        <locus target="#80v"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1301]: It might make sense to state that these are depicted from bottom to top?

**Review by thea-m — CHANGES_REQUESTED** [review]
Very good, Guesh! I made some comments regarding typos. 
And a general comment: I think that you copied the description from the Art Themes, which is fine, but be careful that the Art Themes describe the topic as an abstract, to be valid for all individual renderings it might receive in manuscripts. Here, you describe the concrete rendering of the Art Theme in this manuscript, so instead of describing what the miniatures look like "typically" you can describe how they actually look like here.

**Comment by GueshSol** [comment]
Thank you @thea-m for your excellent remarks, yes I copied the sequence of description from the art themes and just adjusted them how the decorations are distributed in my manuscript in reality, and except for the first phrases after the authority files of the decorations that I added ''typically..'' the rest sequences are arranged in on the basis of the miniatures in the MS.

File: ParisBNF/abb/BNFabb114.xml [hunk 11]
```diff
@@ -913,406 +912,417 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                      </decoNote>
                      <decoNote type="miniature" xml:id="d4">
                         <locus target="#12r"/>
                         <desc> A miniature of the art theme<ref type="authFile"
-                              corresp="AT1109MMWorker"/> with two episodes showing: first the
-                           peasant standing and praying near to the icon<ref type="authFile"
-                              corresp="AT1031VirginAndChild"/> and second, <ref type="authFile"
-                              corresp="AT1031VirginAndChild"/> performing a gesture of blessing up
-                           on the peasant. 
+                              corresp="AT1109MMWorker"/> with one two episodes, showing: first, the
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:943]: "one two episodes" - is it one or two? :)

File: ParisBNF/abb/BNFabb114.xml [hunk 12]
```diff
@@ -913,406 +912,417 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                           in bed; second the sick pilgrim standing before the places to which <persName ref="PRS6819Mary"/>
-                           carries him; third, in varying sequence Golgotha, Jerusalem, the mount of
-                           Olives, Bethlehem and the Jordan are depicted; and fourth,
-                              <persName ref="PRS6819Mary"/> is standing in the middle of the places
-                           depicted with a house scene background.
-                        </desc> 
+                              corresp="AT1116MMJerusalem"/> with four episodes, showing, first, the
+                           sick pilgrim lying in bed; second the sick pilgrim standing before the
+                           places to which <persName ref="PRS6819Mary"/> carries him; third, in
+                           varying sequence Golgotha, Jerusalem, the mount of Olives, Bethlehem and
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1016]: please delete "in varying sequence"

File: ParisBNF/abb/BNFabb114.xml [hunk 13]
```diff
@@ -913,406 +912,417 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d35">
+                        <locus
+                           target="#1v #2r #2v #3rb #4r #4v #5r #5v #6r #6v #7r #13v #22r #28rb #30rb #30v #31r #38r #48rb #53v #54ra #56v #58r 70rb #71b #74rb #75va #76v
+                           #77v #80rb #81r #82rb #82v #83r #83v #84r #84v #85r #85v #86r #86v #87r #87v #88r #88v #89r #89v #90r #90v #100r #100v #101r #101v #108vb #109r   #109v #110r 110v #111r
+                           "/>
+                        <desc> These decorations are stylistically different from the illustrative
+                           miniatures and probaly could be additions by another atrist at some time
+                           after the completion of the first hand textual and decorative compostion
+                           of the amnsucript. They contain images of several saints, martyrs and
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:1311]: manuscript :)

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, wonderful, just some last minor remarks

**Review by thea-m — APPROVED** [review]

**Pull request merged**